package.json

```
@@ -1,7 +1,35 @@
 {
   "name": "stardust-stats-azure",
+  "scripts": {
+    "precommit": "lint-staged"
+  },
   "dependencies": {
     "lodash": "^4.17.11",
     "mongodb": "3.1.13"
+  },
+  "devDependencies": {
+    "husky": "^1.3.1",
+    "lint-staged": "^8.1.5",
+    "prettier": "1.16.4"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js,css,json,md}": [
```

```suggestion
    "*.js": [
```
I suggest to leave only JS